crypto/dso/dso_win32.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include "dso_locl.h"
+#include "e_os.h"
```

Comment:
I've moved e_os.h to be the very first include, before even the system includes.